scheduler/kubernetes/Logmanagement-fluentd and S3.md

```diff
@@ -0,0 +1,305 @@
+       s3_region #S3_REGION#
+       path logs/
+       buffer_path /var/log/px-container/s3
+       s3_object_key_format #indexUUID#_%{path}%{time_slice}_%{index}.%{file_extension}          
+       time_slice_format %Y%m%d%H
+       time_slice_wait 1m
+       utc
+       buffer_chunk_limit 256m
+    </match>    
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: fluentd
+  namespace: kube-system
+  labels:
+    k8s-app: fluentd-logging
+    version: v1
+    kubernetes.io/cluster-service: "true"
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: fluentd-logging
+        version: v1
+        kubernetes.io/cluster-service: "true"
+    spec:
+      serviceAccount: fluentd
+      serviceAccountName: fluentd
+      initContainers:
+      - name: fluentd-init
+        image: hrishi/fluentd-initutils-s3:v1
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
+        command: ['/bin/sh']
+        args: ['-c','/usr/bin/init-fluentd.sh portworx-service']
+        env:
+        - name: "AWS_KEY_ID"
+          valueFrom:
+            secretKeyRef:
+              name: fluentd-px-secrets
+              key: AWS_KEY_ID
+        - name: "AWS_SECRET_KEY_ID"
+          valueFrom:
+            secretKeyRef:
+              name: fluentd-px-secrets
+              key: AWS_SECRET_KEY_ID
+        - name: "S3_BUCKET"
+          valueFrom:
+            secretKeyRef:
+              name: fluentd-px-secrets
+              key: S3_BUCKET
+        - name: "S3_REGION"
+          valueFrom:
+            secretKeyRef:
+              name: fluentd-px-secrets
+              key: S3_REGION
+        volumeMounts:
+        - name: config
+          mountPath: /tmp
+      containers:
+        - name: fluentd
+          image: hrishi/fluentd:v1
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - name: varlog
+              mountPath: /var/log
+            - name: runlog
+              mountPath: /run/log
+            - name: varlibdockercontainers
+              mountPath: /var/lib/docker/containers
+            - name: posloc
+              mountPath: /tmp
+            - name: config
+              mountPath: /fluentd/etc/fluent.conf
+              subPath: fluent.conf
+      volumes:
+        - name: varlog
+          hostPath:
+            path: /var/log
+        - name: runlog
+          hostPath:
+            path: /run/log
+        - name: varlibdockercontainers
+          hostPath:
+            path: /var/lib/docker/containers
+        - name: config
+          configMap:
+            name: fluentd
+        - name: posloc
+          hostPath:
+            path: /mnt
+
+```
+This configuration would enable cluster level logging for the Portworx Pods and publish those logs to an S3 bucket. 
+
+If you would like to setup an Elastic search cluster to ingest the Portworx and the Kubernetes cluster logs please follow the instructions as below. 
```

Comment:
Remove it from this page. We will have a parent page that references this page and the elastic search page.